pkgs/repo_manage/lib/issue_transfer.dart

```diff
@@ -42,13 +42,13 @@ class TransferIssuesCommand extends ReportCommand {
     argParser.addOption(
       'add-label',
       help: 'Add a label to all transferred issues.',
-      valueHelp: 'package:foo',
+      valueHelp: 'package-foo',
```

Comment:
I believe we normally use colons in the package label.

Reply:
I saw that the `labeler.yml` adds `package-foo` labels, so wanted to adapt this. I am completely fine with either

Reply:
Gotcha; we've used `package:foo` in our other monorepos (as far as I'm aware), so I'd like to do the same here.